tutor/src/components/add-edit-question/terms-of-use.js

```diff
@@ -43,9 +43,9 @@ const AddEditQuestionTermsOfUse = observer(({ ux }) => {
   const [agree, setAgree] = useState(false);
   return (
     <StyledAddEditQuestionTermsOfUseModal
-      show={ux.showAddEditTermsOfUse}
+      show={true}
       backdrop="static"
```

Comment:
was the change to be `true` here for debugging?

Reply:
yes! :)